traffic_ops/traffic_ops_golang/deliveryservice/servers/servers.go

```diff
@@ -1023,6 +1027,7 @@ func scanDSInfoRow(row *sql.Row) (DSInfo, bool, error) {
 		}
 		return DSInfo{}, false, fmt.Errorf("querying delivery service server ds info: %v", err)
 	}
+	di.Active = active == tc.DSActiveStateActive
```

Comment:
I am so lost here, why is a boolean being assigned a string and then use a comparison operator?

Reply:
A boolean is not being assigned to a string. A boolean is being assigned to the outcome of comparing two strings. A Delivery Service where `"active": true` in the old data model is equivalent to `"active": "ACTIVE"` in the new data model. So if `active` is exactly `tc.DSActiveStateActive`, then `di.Active` should be `true`. If it's anything else, `di.Active` should be false. Which is what this line does.